packages/completer/src/handler.ts

```diff
@@ -256,7 +261,11 @@ class CompletionHandler implements IDisposable {
     model.setOptions(value.matches || []);
 
     // Update the cursor.
-    model.cursor = { start: value.cursor_start, end: value.cursor_end };
+    const text = this._editor.model.value.text;
```

Comment:
This should be `const text = state.text;`, because the editor can be `null`.  This is why the test was failing.